<launch>

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find ros_diff_drive)/config/diff_drive_cfg.yaml" 
            command="load"/>

  <!-- load the controllers -->
  <node name="controller_spawner" 
        pkg="controller_manager" 
        type="spawner" 
        respawn="false"
	      output="screen" 
        ns="/"
        args="joint_state_controller m2xr_diff_drive_controller"/>

  <!-- convert joint states to TF transforms for rviz, etc -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
	respawn="false" output="screen">
    <!--<remap from="/joint_states" to="/m2xr/joint_states" />-->
  </node>

  <node name="pid_dynamic_reconfigure" pkg="ros_diff_drive" type="SrvDynamicRec.py">
  </node>

</launch>
